e2e/appActions.js

```diff
@@ -112,6 +112,72 @@ async function createDomainObjectWithDefaults(page, { type, name, parent = 'mine
     };
 }
 
+/**
+ * This common function creates a domain object with the default options. It is the preferred way of creating objects
+ * in the e2e suite when uninterested in properties of the objects themselves.
+ *
+ * @param {import('@playwright/test').Page} page
+ * @param {CreateObjectOptions} options
+ * @returns {Promise<CreatedObjectInfo>} An object containing information about the newly created domain object.
+ */
+async function createSineWaveWithInfinityOption(page, { type, name, parent = 'mine' }) {
```

Comment:
It looks like this is mostly the same as `createDomainObjectWithDefaults`. Is it possible to reuse or extend that function here in some way?

Reply:
Yes, I've refactored this. Please take a look.